.github/workflows/generate-and-publish-sdk.yaml

```diff
@@ -1,16 +1,30 @@
+# Copyright 2022 Expedia, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
 name: Generate and Publish SDK
 on:
   workflow_dispatch:
     inputs:
       version:
-        description: 'Generated SDK version'
+        description: 'generated sdk version'
```

Comment:
Why to change the step names?

Reply:
Did not change them, I rather deleted old files and added new ones renamed (removed `experimental` suffix).